libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -60,6 +61,32 @@ export const fetchApplication = createAsyncThunk<Application, { applicationId: s
   }
 )
 
+export const editApplication = createAsyncThunk(
+  'application/edit',
+  async (payload: { applicationId: string; data: Application }) => {
+    const cloneApplication = Object.assign({}, payload.data as any)
```

Comment:
You did this because the api endpoint does not take the full application object right? I think that we should factorize and test this because we are gonna need it everytime we want to update the application. For example, I'm gonna need it to update the storage soon